desktop/cmp/grid/editors/HoistInlineEditorModel.js

```diff
@@ -0,0 +1,144 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {useImperativeHandle} from 'react';
+import composeRefs from '@seznam/compose-react-refs';
+import classNames from 'classnames';
+import {isNil} from 'lodash';
+import {HoistModel, useLocalModel} from '@xh/hoist/core';
+import {bindable, makeObservable} from '@xh/hoist/mobx';
+import {start} from '@xh/hoist/promise';
+import {createObservableRef} from '@xh/hoist/utils/react';
+import './InlineEditors.scss';
+
+/**
+ * A Local Model supporting inline cell editor components in Hoist. Provides the base functionality
+ * needed for supporting inline cell editing in ag-grid and extension points for editors which need
+ * more complex behaviors.
+ *
+ * To create an instance of a component using this model use the hook
+ * {@see useHoistInlineEditorModel}
+ */
+export class InlineEditorModel extends HoistModel {
+    @bindable value;
+
+    ref = createObservableRef();
+
+    /** @member {Column} */
+    column;
+
+    /** @member {GridModel} */
+    gridModel;
+
+    /** @member {Record} */
+    record;
+
+    /** @member {ICellEditorParams} */
+    agParams;
+
+    get startedEdit() {
+        return this.agParams?.cellStartedEdit;
+    }
+
+    /** @returns {HoistInputModel} */
+    get inputModel() {
+        return this.ref.current;
+    }
+
+    get inputEl() {
+        return this.inputModel?.inputRef.current;
```

Comment:
HoistInputModel has an `inputEl` getter that does a bit more work for you and can be delegated to.